🐛 Destination MSSQL: fix issue with unicode symbols handling (ex.\u2028 showed as ?)

etsybaev opened the pull request:
## What
https://github.com/airbytehq/airbyte/issues/3552 MSSQL Destination: Fix newline handling

## How
Fixed MSSQL _airbyte_data field's type to support unicode.

This is for Microsoft SQL Server:
**NVARCHAR** is Unicode - 2 bytes per character, therefore max. of 1 billion characters; will handle East Asian, Arabic, Hebrew, Cyrillic etc. characters just fine.
**VARCHAR** is non-Unicode - 1 byte per character, max. capacity is 2 billion characters, but limited to the character set you're SQL Server is using, basically - no support for those languages mentioned before


## Pre-merge Checklist
- [x] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

sherifnada commented on airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java:
```diff
@@ -53,7 +53,7 @@ public String createTableQuery(String schemaName, String tableName) {
             + "WHERE s.name = '%s' AND t.name = '%s') "
             + "CREATE TABLE %s.%s ( \n"
             + "%s VARCHAR(64) PRIMARY KEY,\n"
-            + "%s VARCHAR(MAX),\n"
```
@etsybaev can you leave a comment in the code describing why NVARCHAR is used since we typically use VARCHAR?

etsybaev replied:
@sherifnada done. Added comment to code and documentation. Thanks

sherifnada reviewed (approved):
Thanks for also updating the docs! 

Once you've addressed the one comment, please bump the connector version like described here and update your PR with the new version: https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector

then publish the connector like described in the link.

ChristopheDuong commented on airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java:
```diff
@@ -53,7 +53,7 @@ public String createTableQuery(String schemaName, String tableName) {
             + "WHERE s.name = '%s' AND t.name = '%s') "
             + "CREATE TABLE %s.%s ( \n"
             + "%s VARCHAR(64) PRIMARY KEY,\n"
-            + "%s VARCHAR(MAX),\n"
```
Normalization is not handling MSSQL yet (so nothing to do there, unless implementing it...), otherwise I don't know how the mssql-dbt adapter plugin handle strings atm

etsybaev commented:
Hi @sherifnada.
I've updated versions but found it and updated in 2 places. Not sure if I also had to update it in airbyte-config/init/src/main/resources/seed/destination_definitions.yaml ?
Could you please also clarify what is the workflow for pushing images? the command "./tools/bin/check_images_exist.sh" works for me locally, but obviously failed in the build for this PR as the image has not been released globally. What are the next steps from my side? 
Many thanks in advance!

sherifnada commented:
@etsybaev to publish an image you need to: 
1. _before_ bumping the image version, have a passing build and PR approvals
2. bump the image in Dockerfile, YAML, and JSON config files. The build will fail -- it's OK because we know the only change we made was version bump. 
3. Run the `/publish` command
4. once publish is successful, merge to master

etsybaev commented:
/test connector=connectors/destination-mssql

etsybaev commented:
/publish connector=connectors/destination-mssql
